modules/core/src/main/java/org/locationtech/jts/coverage/CoverageSimplifier.java

```diff
@@ -64,7 +72,20 @@ public static Geometry[] simplify(Geometry[] coverage, double tolerance) {
     CoverageSimplifier simplifier = new CoverageSimplifier(coverage);
     return simplifier.simplify(tolerance);
   }
-  
+
+  /**
+   * Simplifies the boundaries of a set of polygonal geometries forming a coverage,
+   * preserving the coverage topology.
+   *
+   * @param coverage a set of polygonal geometries forming a coverage
+   * @param tolerances the simplification tolerances for each coverage
+   * @return the simplified polygons
+   */
+  public static Geometry[] simplify(Geometry[] coverage, List<Double> tolerances) {
```

Comment:
Why is tolerances a List and not an array?  It seems more uniform for it to have the same structure as the geometries.